UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -973,49 +973,77 @@ private void OnDrawGizmos()
 	/// </summary>
 	public string Examine(Vector3 worldPos)
 	{
-		var healthFraction = OverallHealth / maxHealth;
-		var healthString = "";
+		if (this is PlayerHealth)
+		{
+			// Let ExaminablePlayer take care of this.
+			return default;
+		}
 
-		if (!IsDead)
+		return GetExamineText();
+	}
+
+	public string GetExamineText()
+	{
+		// Assume animal
+		string theyPronoun = "It";
+		string theirPronoun = "its";
+
+		var cs = GetComponentInParent<PlayerScript>()?.characterSettings;
+		if (cs != null)
+		{
+			theyPronoun = cs.TheyPronoun();
+			theyPronoun = theyPronoun[0].ToString().ToUpper() + theyPronoun.Substring(1);
+			theirPronoun = cs.TheirPronoun();
+		}
+
+		var healthString = $"{theyPronoun} is ";
+		if (IsDead)
 		{
+			healthString += "limp and responsive; there are no signs of life";
```

Comment:
limp and responsive? :)
```suggestion
			healthString += "limp and unresponsive; there are no signs of life";
```